_posts/2016-11-09-foreman-community-newsletter-january-2016.md

```diff
@@ -0,0 +1,71 @@
+---
+
+### Foreman 1.10 is out
+Foreman 1.10 was released on 23rd of December 2015 making it an early Christmas present. The headline features include [orchestration rebuilder](https://www.youtube.com/watch?v=MxPtzWhiE1o) for rebuilding hosts' DHCP, DNS and TFTP records, [global statuses](https://www.youtube.com/watch?v=2FNLwf5Z47A) for hosts, host and hostgroup cloning improvements, support for [unsetting attributes](https://www.youtube.com/watch?v=eXRiCYjmXBk), enhanced DNS plugin support and [parameter improvements](https://www.youtube.com/watch?v=hXl70osESlQ). Full release notes can be found [here](http://theforeman.org/manuals/1.10/index.html#Releasenotesfor1.10).
+
+### Katello 2.4 is out
+Katello 2.4 which brings a number of improvements was released on 5th of January 2016, shortly after Foreman's 1.10 release. This release provides compatibility with Foreman 1.10, migrates most objects from Elasticsearch to scoped_search and lots of improvements and bugxies. Check out the [release notes](http://www.katello.org/docs/2.4/release_notes/release_notes.html) for a full list of updates.
+
+### New and updated plugins since last December's newsletter
+  - [foreman-tasks](https://github.com/theforeman/foreman-tasks) updated to 0.7.10 on 7th of January 2016
+  - [foreman_setup](https://github.com/theforeman/foreman_setup) updated to 3.1.0 on 6th of January 2016
+  - [foreman_templates](https://github.com/theforeman/foreman_templates) updated to 2.0.3 on 6th of January 2016
+  - [katello-installer](https://github.com/Katello/katello-installer) updated to 2.4.0 on 23rd of December 2015
+  - [foreman-selinux](https://github.com/theforeman/foreman-selinux) updated to 1.10.0 on 23rd of December 2015
+  - [foreman-installer](https://github.com/theforeman/foreman-installer) updated to 1.10.0 on 23rd of December 2015
+  - [smart-proxy](https://github.com/theforeman/smart-proxy) updated to 1.10.0 on 23rd of December 2015
+  - [smart_proxy_remote_execution_ssh](https://github.com/theforeman/smart_proxy_remote_execution_ssh) updated to 0.0.10 on 22nd of December 2015
+  - [foreman-docker](https://github.com/theforeman/foreman-docker) updated to 2.0.0 on 18th of December 2015
+  - [bastion](https://github.com/Katello/bastion) updated to 2.1.0 on 18.12.2016
+  - [foreman-one](https://github.com/theforeman/foreman-one) updated to 0.4 on 18th of December 2015
+  - [smart_proxy_chef](https://github.com/theforeman/smart_proxy_chef) updated to 0.1.6 on 18th of December 2015
+  - [chef-handler-foreman](https://github.com/theforeman/chef-handler-foreman) updated to 0.1.1 on 18th of December 2015
+  - [hammer_cli_foreman_remote_execution](https://github.com/theforeman/hammer_cli_foreman_remote_execution) updated to 0.0.2 on 17th of December 2015
+  - [foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution) updated to 0.1.2 on 17th of December 2015
+  - [hammer-cli](https://github.com/theforeman/hammer-cli) updated to 0.5.1 on 15th of December
```

Comment:
`hammer-cli` is not plugin. `hammer-cli-*` are hammer plugins, not Foreman plugins and it seems confusing to mix them together.